src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -386,6 +386,15 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 		}
 	}
 
+	// Unlabel run-level from assisted-installer namespace after the installation
+	data := []byte(`{"metadata":{"labels":{"$patch": "delete", "openshift.io/run-level":"0"}}}`)
+	c.log.Infof("Removing run-level label from %s namespace", c.ControllerConfig.Namespace)
+	err = c.kc.PatchNamespace(c.ControllerConfig.Namespace, data)
+	if err != nil {
+		c.log.WithError(err).Warn("Failed to unlabel AI namespace after the installation.")
+		return err
```

Comment:
What do you think about adding a docstring with something like:

```
Keeping the `run-level` label represents a security risk, and it overwrites the SecurityContext configurations used for applications deployed in this namespace. Because of this, failing to remove the `run-level` label is considered a deployment failure, hence the `return err`
```

Feel free to rephrase it :)

Reply:
Let's maybe not handle the error here. If the patch command for whatever reason fail, we should not stop the workflow on this specific case... at least until we have a proper solution.

Reply:
Changed, the error is not handled so that the logic we are adding has no chance of failing the cluster deployment